src/AutoRest.CSharp/LowLevel/Generation/LowLevelExampleComposer.cs

```diff
@@ -831,7 +839,13 @@ private static IReadOnlyList<MethodSignatureBase> GetClientInvocationChain(LowLe
 
                 client = client.ParentClient;
             }
-            callChain.Push(client.SecondaryConstructors.Where(c => c.Modifiers == MethodSignatureModifiers.Public).OrderBy(c => c.Parameters.Count).First());
+            var ctor = client.SecondaryConstructors.Where(c => c.Modifiers.HasFlag(MethodSignatureModifiers.Public)).OrderBy(c => c.Parameters.Count).FirstOrDefault();
+
+            // if we do not have a public secondary constructor, we have to return null and compose nothing
+            if (ctor == null)
+                return null;
```

Comment:
We can only return null here because to demo how to use a method on a client, the first step is to create a client instance, but if all the constructors are changed to internal, we are not able to write any example (if we do not change the logic here)